CONTRIBUTING.md

```diff
@@ -26,6 +26,9 @@ author:
   name: OpenFisca  # 120 characters max
   contact: contact@openfisca.org  # this email address will only be used by the Core team for questions related to the reuse
   link: https://openfisca.org  # optional URL of the author organisation
+source:
+  repository: https://github.com/openfisca/legislation-explorer # optional URL
+  license: AGPL-3.0-only # optional SPDX license identifier, see https://spdx.org/licenses/
```

Comment:
I prefer we don't insist on the “optional” part and instead focus on the “proprietary” option below as an alternative, but consider it a given that contributors should provide their source 🙂 

```suggestion
  repository: https://github.com/openfisca/legislation-explorer  # URL of the source code
  license: AGPL-3.0-only  # SPDX license identifier, see https://spdx.org/licenses/
```